xml/System.Text.Json.Serialization/ReferenceHandler`1.xml

```diff
@@ -23,7 +23,7 @@
   <Interfaces />
   <Docs>
     <typeparam name="T">To be added.</typeparam>
```

Comment:
```suggestion
    <typeparam name="T">The type of the <see cref="ReferenceResolver"/> to create on each serialization or deserialization call.</typeparam>
```

Looks like this wasn't ported, perhaps there is a bug in the DocsPortingTool @carlossanlop?